source/extensions/clusters/strict_dns/strict_dns_cluster.h

```diff
@@ -74,6 +77,7 @@ class StrictDnsClusterImpl : public BaseDynamicClusterImpl {
   Network::DnsLookupFamily dns_lookup_family_;
   uint32_t overprovisioning_factor_;
   bool weighted_priority_health_;
+  Random::RandomGeneratorImpl random_generator_;
```

Comment:
we should always use the random generator from factory context.

Reply:
@wbpcode Where is the random generator from the factory context? I'm looking in `cluster_factory.h` and I see that it includes `random_generator.h`, but I don't see any usages of a random generator?

Reply:
nevermind I found it.